.vscode/spellright.dict

```diff
@@ -0,0 +1,250 @@
+_cli
+cli
+db
+Postgres
+SQLALCHEMY
+postgresql
+env
+ENV
+urlsafe
+cfg
+Quickstart
+MWh
+todo
+DSO
+CO₂
+―
+USEF
+sha256_crypt
+WAPE
+int
+ci
+webargs
+Cors
+importlib
+timetomodel
+CDF
+forecastiopy
+utils
+Dockerization
+dotenv
+sys
+os
+WSGI
+hosters
+Pyomo
+Cbc
+cbc
+coinor
+redis
+Redis
+flexdemo
+MAPBOX
+mapbox
+timedelta
+datetimes
+RQ
+cplex
+glpk
+FLEXMEASURES_PLUGIN_PATH
+plugin
+fmplugin
+init
+py
+Jinja2
+'fmplugin'
```

Comment:
fmplugin is in the list twice, this one with quotations should be removed I think

Reply:
Agreed!

Just for the sake of curiosity, I tried with the words `prueba`,`"prueba"` and `'prueba'` to see how the are included in the spellright.dict in me version. In either case, the literal `prueba` is added.